src/routes/LoadSafePage/steps/LoadSafeAddressStep.tsx

```diff
@@ -103,15 +104,16 @@ function LoadSafeAddressStep(): ReactElement {
   return (
     <Block className={classes.padding} data-testid={'load-safe-address-step'}>
       <Block margin="md">
-        <Paragraph color="primary" noMargin size="md">
-          You are about to add an existing Gnosis Safe. First, choose a name and enter the Safe address. The name is
-          only stored locally and will never be shared with Gnosis or any third parties.
-          <br />
+        <Paragraph color="primary" noMargin size="lg">
+          You are about to add an existing Gnosis Safe on <NetworkLabel /> first, choose a name and enter the Safe
```

Comment:
You need a full stop after the label.

Reply:
I can also see that the network label renders a paragraph:
```
      <StyledParagraph network={selectedNetwork} size="xs">
        {selectedNetwork.label}
      </StyledParagraph>
```
and a paragraph cannot be nested in another paragraph: https://stackoverflow.com/a/4291515/7820085

which is currently the case